### PR TITLE
Pill 1: edit for style and clarity

### PR DESCRIPTION
**Pull request by michaelpj**
I thought I'd have a go at going through the pills to make them a little more stylistically formal (since they're now at least semi-official), and improve the clarity in a few places. I've aimed to leave the content largely unchanged.

I've just done the first one to see if it's desirable - if so I'll have a look at doing the others too.

### CODE REVIEW

File: pills/01-why-you-should-give-it-try.xml [hunk 1]
```diff
@@ -10,50 +10,50 @@
...
+      Nix is a purely functional package manager and deployment
+      system for POSIX.
     </para>
 
     <para>
-      The purpose of this post is to convince you to give Nix a try.
+      There's a lot of documentation that describes what Nix, <link
+      xlink:href="https://nixos.org/nixos">NixOS</link> and related
+      projects are.
+      But the purpose of this post is to convince you to give Nix a try.
```
Comment (grahamc) [inline pills/01-why-you-should-give-it-try.xml]: Did you want this to be a new para, or just trying to not disrupt the formatting of the rest of the lines?

File: pills/01-why-you-should-give-it-try.xml [hunk 2]
```diff
@@ -10,50 +10,50 @@
...
-      papers</link>, <link
-      xlink:href="http://www.infoq.com/articles/configuration-management-with-nix">articles</link>
-      and wiki are excellent in explaining how Nix/NixOS works, how
-      you can use it and the amount of cool things being done with it,
-      however sometimes you may feel some magic happening behind the
-      scenes hard to grasp in the beginning.
+      The <link xlink:href="https://nixos.org/nix/manual/">Nix</link>,
+      <link xlink:href="https://nixos.org/nixpkgs/manual/">Nixpkgs</link>, and
+      <link xlink:href="https://nixos.org/nixos/manual/">NixOS</link> manuals;
+      <link xlink:href="http://nixos.org/docs/papers.html">related papers</link>;
```
Comment (grahamc) [inline pills/01-why-you-should-give-it-try.xml]: This papers link is dead :(

File: pills/01-why-you-should-give-it-try.xml [hunk 3]
```diff
@@ -122,89 +121,89 @@
...
      <para>
        Nix makes no assumptions about the global state of the system.
        This has many advantages, but also some drawbacks of course.
-       The core of a Nix based system is the Nix store, usually
-       installed under /nix/store, and some tools to manipulate the
-       store. In Nix there's the notion of a derivation rather than a
-       package. The difference might be subtle at the beginning, so I
-       will often mix both of the words, ignore that.
+       The core of a Nix system is the Nix store, usually
+       installed under <literal>/nix/store</literal>, and some tools to manipulate the
```
Comment (grahamc) [inline pills/01-why-you-should-give-it-try.xml]: If you want to go Ultimate Docbookery you could use `<filename>` or even `<filename class="directory">`, but this is by no means necessary :)

File: pills/01-why-you-should-give-it-try.xml [hunk 4]
```diff
@@ -122,89 +121,89 @@
...
+       where the hash uniquely identifies the derivation (this isn't quite true,
+       it's a little more complex), and the name is the name of
        the derivation.
      </para>
      <para>
-       Let's take the bash derivation as an example:
+       Let's take a bash derivation as an example:
        <literal>/nix/store/s4zia7hhqkin1di0f187b79sa2srhv6k-bash-4.2-p45/</literal>.
-       It is a directory that contains <literal>bin/bash</literal>.
+       This picks out a directory that contains <literal>bin/bash</literal>.
```
Comment (grahamc) [inline pills/01-why-you-should-give-it-try.xml]: This new wording is more confusing than the previous versionn

**Comment by grahamc** [comment]
Really well done. A few bits of feedback, but this is much improved already! Merging. If you send PRs fixing my feedback, I'll merge those too!

File: pills/01-why-you-should-give-it-try.xml (hunk 1 above)
Comment (michaelpj) [reply pills/01-why-you-should-give-it-try.xml]: The latter.

File: pills/01-why-you-should-give-it-try.xml (hunk 2 above)
Comment (michaelpj) [reply pills/01-why-you-should-give-it-try.xml]: Hm, I'll remove it for now. I note that there's a `papers-in.html` at `nixos.org/docs` - possibly something has gone wrong with the site generation there?

**Comment by michaelpj** [comment]
Great, I'll maybe have a look at a few more pills this weekend.

**Comment by Profpatsch** [comment]
Awesome work.

**Pull request merged**